### PR TITLE
flow: 0.22.1 -> 0.28.0

### PR DESCRIPTION
**Pull request by despairblue**
###### Motivation for this change

I needed an up to date version.
###### Things done
- [x] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
  **No idea how.**
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
  **No idea how. I run `nix-env -i flow -f ../../../../../default.nix` to build and install it.**
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

I'd be happy to fulfill the other tasks, I just need some guidance.

### CODE REVIEW

**Comment by despairblue** [comment]
@puffnfresh @globin

File: pkgs/development/tools/analysis/flow/default.nix [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
     repo = "flow";
     rev = "v${version}";
-    sha256 = "11d04g8rvjv2q79pmrjjx8lmmm1ix8kih7wc0adln0ap5123ph46";
+    sha256 = "1xryv1366zc385r82r6n832xkaqcm63zs1baizl02qchfzfa3am2";
   };
 
+  patchPhase = ''
+    echo "HERE"
+    git --version
+  '';
```
Comment (fpletz) [inline pkgs/development/tools/analysis/flow/default.nix]: Do you really need that `patchPhase` or is it a leftover from debugging?

Comment (despairblue) [inline pkgs/development/tools/analysis/flow/default.nix]: @fpletz Good catch. I left it there accidentally. I'll remove it.

Comment (despairblue) [inline pkgs/development/tools/analysis/flow/default.nix]: @fpletz done

**Comment by globin** [comment]
Thanks

**Pull request merged**